src/crates/sources/github.rs

```diff
@@ -69,17 +70,23 @@ impl List for GitHubList {
 pub struct GitHubRepo {
     pub org: String,
     pub name: String,
+    pub sha: Option<String>,
 }
 
 impl GitHubRepo {
     pub(crate) fn slug(&self) -> String {
-        format!("{}/{}", self.org, self.name)
+        if let Some(ref sha) = self.sha {
+            format!("{}/{}/{}", self.org, self.name, sha)
+        } else {
+            format!("{}/{}", self.org, self.name)
+        }
```

Comment:
Let's not change this: the slug of a repo doesn't contain the SHA.

Reply:
Ok, changed